.github/workflows/maven-verify.yml

```
@@ -0,0 +1,36 @@
+name: Maven Verify
+
+on:
+  pull_request:
+    branches:
+      #      - main
```

```suggestion
      - main
```
Why is this commented out?